plio/models.py

```diff
@@ -140,3 +143,18 @@ class Question(SafeDeleteModel):
 
     class Meta:
         db_table = "question"
+
+
+@receiver(post_save, sender=Plio)
+@receiver(post_delete, sender=Plio)
+def plio_update_cache(sender, instance, **kwargs):
+    invalidate_cache_for_instance(instance)
+
+
+@receiver(post_save, sender=Video)
+@receiver(post_delete, sender=Video)
+def video_update_cache(sender, instance, **kwargs):
+    # fetch all plio with video id
+    plios = Plio.objects.filter(video_id=instance.id)
+    # invalidate saved cache for the plios
+    invalidate_cache_for_instances(plios)
```

Comment:
need to do this for question, item, imagw as well no?

Reply:
Right! I'll update.